elastic/apm-server
# systemtest: disable fleet ID verification pipeline

## Motivation/summary

Disable the pipeline so `event.agent_id_status` and `event.ingested` are not set by the Fleet final pipeline. We explicitly set `event.ingested` in `traces-apm.sampled` for tail-based sampling's internal usage.

## Checklist

~- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/main/CHANGELOG.asciidoc)~
~- [ ] Documentation has been updated~

## How to test these changes

Run system tests with and without the `xpack.fleet.agentIdVerificationEnabled` change to `testing/docker/kibana/kibana.yml`. Ignoring the approval diffs, the tests should all pass.

## Related issues

https://github.com/elastic/elasticsearch/pull/97546
